client/src/components/QA/QuestionsAndAnswers.jsx

```diff
@@ -33,9 +45,12 @@ const QuestionsAndAnswers = () => {
         <button className="qa-search-btn">Search</button>
       </form>
 
-      <div data-testid="qa"><QACardQuestions questions={questions} /></div>
+      <div data-testid="qa"><QACardQuestions questions={questionsToShow} /></div>
 
-      <button className="qa-more">MORE ANSWERED QUESTIONS</button>
+      <button className="qa-more" onClick={() => {
```

Comment:
This is a really clean and simple way to slice the array for viewing the list. I'll implement this on my relevant components as well.